hs_core/hydroshare/resource.py

```diff
@@ -722,6 +722,7 @@ def add_resource_files(pk, *files, **kwargs):
     else:
         if resource.resource_type == "CompositeResource" and auto_aggregate:
             utils.check_aggregations(resource, ret)
+        utils.resource_modified(resource, user, overwrite_bag=False)
```

Comment:
Since we are setting the resource flag here, I think we don't need it here (potentially save few db queries):
https://github.com/hydroshare/hydroshare/blob/5135-dirty-bag-toggle/hs_core/hydroshare/utils.py#L996

Reply:
good call, removed it, thanks!